components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/publisher/ApplicationAuthorizedAPIManagementEventPublisherProxy.java

```diff
@@ -0,0 +1,132 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+package org.wso2.carbon.identity.application.mgt.publisher;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.application.common.IdentityApplicationManagementException;
+import org.wso2.carbon.identity.application.mgt.internal.ApplicationManagementServiceComponentHolder;
+import org.wso2.carbon.identity.event.IdentityEventConstants;
+import org.wso2.carbon.identity.event.IdentityEventException;
+import org.wso2.carbon.identity.event.event.Event;
+import org.wso2.carbon.identity.event.services.IdentityEventService;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class handles creating event and publishing events related to application authorized api management.
+ */
+public class ApplicationAuthorizedAPIManagementEventPublisherProxy {
+
+    private static final Log log = LogFactory.getLog(ApplicationAuthorizedAPIManagementEventPublisherProxy.class);
+    private static final ApplicationAuthorizedAPIManagementEventPublisherProxy proxy =
+            new ApplicationAuthorizedAPIManagementEventPublisherProxy();
+
+    private ApplicationAuthorizedAPIManagementEventPublisherProxy() {
+
+    }
+
+    public static ApplicationAuthorizedAPIManagementEventPublisherProxy getInstance() {
+
+        return proxy;
+    }
+    public void publishPreUpdateAuthorizedAPIForApplication(String appId, String apiId,
```

Comment:
as all the parameters are started in a new line start the variable definition String apiId in a new line

Reply:
Updated